node/src/Pos/Delegation/Logic/Mempool.hs

```diff
@@ -228,10 +227,7 @@ processProxySKHeavyInternal psk = do
         coherent <- uses dwTip $ (==) dbTipHash
         dwMessageCache %= LRU.insert msg curTime
         let doublePosted = alreadyPosted && not isRevoke
-        let maxMemPoolSize = memPoolLimitRatio * maxBlockSize
-            -- Here it would be good to add size of data we want to insert
-            -- but it's negligible.
-            exhausted = memPoolSize >= maxMemPoolSize
+            exhausted = memPoolSize >= maxBlockSize
```

Comment:
Again, I think there's a confusion of units. And shouldn't we getting the pool size from the constants, rather than based on how many transactions can fit in a block.